src/services/SalesEmployeeService.ts

```diff
@@ -0,0 +1,71 @@
+import axios, { AxiosResponse } from "axios";
+import { SalesEmployeeResponse } from "../models/SalesEmployeeResponse";
+import { SalesEmployeeRequest } from "../models/SalesEmployeeRequest";
+
+export const getSalesEmployees = async (): Promise<SalesEmployeeResponse[]> => {
+    try {
+        const response: AxiosResponse = await axios.get("http://localhost:8080/api/sales-employee");
+
+        return response.data;
+    } catch (e) {
+        throw new Error('Failed to get sales employees');
+    }
+}
+
+export const getSalesEmployee = async (id: String): Promise<SalesEmployeeResponse[]> => {
```

Comment:
id as int?

Reply:
is a string in the url, saves an extra try catch for unable to parse int error